src/vault/client/ext/aws.clj

```diff
@@ -59,10 +63,10 @@
 (defn signer
   "Signing object used to perform AWS4 signing on sts request object."
   ^AWS4Signer
-  []
+  [region]
   (doto (AWS4Signer.)
     (.setServiceName "sts")
-    (.setRegionName "us-east-1")))
```

Comment:
This PR makes sense to me. I don't remember perfectly, but I think this was hard-coded because I was under the impression that all of these STS requests go to us-east-1. This might have been true/documented at one point and is now out-dated?

Reply:
I think this was only true [pre-2015](https://aws.amazon.com/blogs/security/aws-security-token-service-sts-is-now-active-by-default-in-all-aws-regions/).

I'm guessing that we experimentally determined that all our STS requests were going to `us-east-1` because that's the "global" region, and we had to set a region here so that's what we used.